### PR TITLE
Or 1203 titond backend k8s secret generate new wallet blocksigner relayer sequencer proposer

### PR DESCRIPTION
**Pull request by ohbyeongmin**
create account : blocksigner, relayer, sequencer, proposer

### CODE REVIEW

File: pkg/api/component.go [hunk 1]
```diff
@@ -51,3 +62,47 @@ func (t *TitondAPI) DeleteComponentById(componentID uint) error {
...
+
+func (t *TitondAPI) checkNamespace(namespace string) bool {
+	_, err := t.k8s.GetNamespace(namespace)
+	if err != nil {
+		return false
+	}
+	return true
+}
+
+func (t *TitondAPI) createAccounts(namespace string) error {
```
Comment (rlgns98kr) [inline pkg/api/component.go:86]: 😍

File: go.mod [hunk 2]
```diff
@@ -5,6 +5,8 @@ go 1.20
 require (
 	github.com/DATA-DOG/go-sqlmock v1.5.0
 	github.com/aws/aws-sdk-go v1.47.9
+	github.com/emicklei/go-restful/v3 v3.9.0
```
[the remark this answers is not in the document]
Comment (ohbyeongmin) [reply go.mod]: i fixed it! thank you

File: pkg/api/component.go [hunk 3]
```diff
@@ -3,17 +3,28 @@ package api
...
 	namespace := generateNamespace(component.NetworkID)
-	_, err := t.k8s.GetNamespace(namespace)
-	if err != nil {
-		t.k8s.CreateNamespace(namespace)
+
+	if !t.checkNamespace(namespace) {
+		if _, err := t.k8s.CreateNamespace(namespace); err != nil {
+			return nil, err
+		}
+		if err := t.createAccounts(namespace); err != nil {
```
[the remark this answers is not in the document]
Comment (rlgns98kr) [reply pkg/api/component.go:24]: we can make both namespace and accounts when creating network. 🙇🏼‍♂️

[the remark this answers is not in the document]
Comment (ohbyeongmin) [reply pkg/api/component.go:24]: Yes! I also considered creating a namespace and accounts after the network was created. i will fix quickly! thank you

[the remark this answers is not in the document]
Comment (ohbyeongmin) [reply pkg/api/component.go]: i fixed it!

**Review by rlgns98kr — APPROVED** [review]
Thank you very much! 😍

**Pull request merged**